.gitignore

```diff
@@ -31,6 +33,8 @@ __pycache__/
 
 # Temporary files or directories
 tags
+_*.py
```

Comment:
(It is relatively normal for private python modules to be named `_filename.py`, and relatively uncommon to dynamically generate *valid* (importable) python modules.  So ignoring `_filename.py` is a little bit strange.  Maybe ignoring them only under a particular directory is better)

Reply:
I added this because Matteo's script generates `_filename.py`. I'll simply remove this from the gitignore. I don't think fortran has any need to output this python script, but not my call

Reply:
One can add a command line option to disable the output of the python script.